sync/src/main/kotlin/gropius/sync/AbstractSync.kt

```diff
@@ -150,6 +150,18 @@ abstract class AbstractSync(
         imsProject: IMSProject, issueId: String, newState: IssueState, users: List<User>
     ): TimelineItemConversionInformation?
 
+    /**
+     * Incorporate a templated field change
+     * @param imsProject IMS project to sync
+     * @param issueId GitHub ID of the issue
```

Comment:
why is there a GitHub id in the AbstractSync?

Reply:
62aa13dffbc73e4f6bd19814988eb3b515342a08